api/go.mod

```diff
@@ -25,7 +26,6 @@ require (
 	github.com/hashicorp/errwrap v1.1.0 // indirect
 	github.com/hashicorp/go-immutable-radix v1.3.1 // indirect
 	github.com/hashicorp/go-msgpack v0.5.5 // indirect
-	github.com/hashicorp/go-multierror v1.1.1 // indirect
```

Comment:
`make go-mod-tidy` fixed this, which is weird b/c I thought that we had a CI job to ensure everything is always tidy.